release-team/role-handbooks/communications/README.md

```diff
@@ -114,6 +114,10 @@ It helps to work closely with the Release Lead and use the respective SIG Slack
 
 When a placeholder PR is created, make sure the PR has been opened using a `draft:true` parameter in its metadata. Feature blogs should not be merged **until after the release date and the comms team has assigned a publication date**.
 
+Once the placeholder PRs are open, **assign a writing buddy to each PR**. Select the writing buddy from the KEP authors of the other open placeholder PRs and pair them up. The writing buddy will act as a peer reviewer and provide support for the main author of the feature blog.
```

Comment:
Consider saying not every Feature Blog needs a writing buddy;
- Some are written collaboratively within SIGs (an example from this cycle is the DRA blog)
- Some are opened with the blog content already done (See [Slack suggestion](https://kubernetes.slack.com/archives/CNT9Y603D/p1750923294315749))